godump.go

```diff
@@ -107,20 +190,25 @@ func DumpHTML(vs ...any) string {
 	enableColor = true
 
 	var sb strings.Builder
-	sb.WriteString(`<body style='background-color:black;'><pre style="background-color:black; color:white; padding:5px; border-radius: 5px"></body>` + "\n")
+	sb.WriteString(`<body style='background-color:black;'><pre style="background-color:black; color:white; padding:5px; border-radius: 5px">` + "\n")
 
 	tw := tabwriter.NewWriter(&sb, 0, 0, 1, ' ', 0)
 	printDumpHeader(&sb, 3)
-	writeDump(tw, vs...)
+	d.writeDump(tw, vs...)
 	tw.Flush()
 
-	sb.WriteString("</pre>")
+	sb.WriteString("</pre></body>")
 	return sb.String()
 }
 
 // Dd is a debug function that prints the values and exits the program.
 func Dd(vs ...any) {
-	Dump(vs...)
+	defaultDumper.Dd(vs...)
+}
+
+// Dd is a debug function that prints the values and exits the program.
+func (d *Dumper) Dd(vs ...any) {
```

Comment:
I wouldn't add Dd to Dumper for now.

While Dd was somehow a short name and operator. I feel like DumpExit would be a better name.

So for now I wouldn't add it. It would be impossible to remove it later.

Reply:
`Dd` is a 1:1 of Laravel's (Dump and Die) shorthand https://laravel.com/docs/12.x/collections#method-dd

If we're to make a DumpExit we should also have a `Dd` to keep things consistent